packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -580,7 +622,9 @@ export default decorate([
             get(() => hostsById[poolsById[$container].master].version)
         ),
       srPredicate: ({ srs }) => sr => isSrWritable(sr) && !includes(srs, sr.id),
-      remotePredicate: ({ remotes }) => ({ id }) => !includes(remotes, id),
+      remotePredicate: ({ proxyId, remotes }) => remote =>
+        remotes.indexOf(remote.id) === -1 &&
+        (proxyId == null || remote.value.proxy === proxyId),
```

Comment:
I don't understand this predicate. Please explain to me ^^

Reply:
If a proxy is selected, the user can only choose remotes which are linked to this proxy.